api/comments_api.py

```diff
@@ -45,8 +45,9 @@ def _should_show_comment(
     """Check whether a comment should be visible to the user."""
     return comment.deleted_by is None or email == comment.deleted_by or is_admin
 
-  def do_get(self, feature_id: int,
-      field_id: Optional[int]=None) -> dict[str, dict[str, Any]]:
+  def do_get(self, **kwargs) -> dict[str, dict[str, Any]]:
```

Comment:
I think the return type should be something more like
dict[str, list[dict[str, Any]]]

Reply:
You're right - I think this is part of the original change I made to introduce type hinting, but with no evaluation of whether or not it was correct with mypy - this should be changed in the mypy PR as well